bindata/cloud-network-config-controller/controller.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
no replicas here in this deployment but you have leader election code in https://github.com/openshift/cloud-network-config-controller/pull/6

Why have LE if theres only going to be one instance of ```cloud-network-config-controller``` ?

Reply:
For the same reason most operators have a leader election defined: on upgrades in certain corner cases there can be situations where one replica of a deployment is started on one node while the previous replica on the old node (the one being rebooted) still runs, it's thus just used as redundancy mechanism to avoid running two instances of the same controller at any point in time.